src/main/kotlin/no/nav/klage/oppgave/service/BehandlingService.kt

```diff
@@ -56,6 +57,22 @@ class BehandlingService(
         return behandling
     }
 
+    fun setSattPaaVent(
+        behandlingId: UUID,
+        setNull: Boolean = false,
```

Comment:
Jeg er ikke helt sikker på denne, eller om det burde vært to ulike metoder, men dette funker jo :) Vi kjør på det.